agent/consul/grpc_integration_test.go

```diff
@@ -73,12 +52,66 @@ func TestGRPCIntegration_ConnectCA_Sign(t *testing.T) {
 		Service:    "foo",
 	})
 
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	t.Cleanup(cancel)
+
+	ctx = public.ContextWithToken(ctx, TestDefaultInitialManagementToken)
```

Comment:
Adding a token to the context because the new testGRPCIntegrationServer turns ACLs on.